context/requirements.txt

```
@@ -4,4 +4,6 @@ requests==2.22.0
 pyyaml==5.3.1
 python-datauri==0.2.8
 python-frontmatter==0.5.0
+vitessce==0.1.0a8
+pandas==1.1.2
```

```suggestion
pandas==1.1.2 # TODO: remove: https://github.com/vitessce/vitessce-python/issues/51
```
If it raises questions for the reviewer, it may raise questions for someone reading it in the future.